scripts/xcode_select_current_version.sh

```diff
@@ -0,0 +1,3 @@
+# Select the current officially supported version of Xcode with the naming convention used by GitHub Actions/Azure DevOps Build Pipelines
```

Comment:
If we're just copying images do we need Xcode at all?

Reply:
Nope, forgot to remove it.